src/api/dependenciesConfig.js

```diff
@@ -87,6 +88,12 @@ export type DependencyConfig = {|
   // in new instances.)
   // Defaults to false if unset.
   +autoActivateOnIdentityCreation?: boolean,
+
+  // If this is set, then a default time period will be injected in the
+  // dependency config with the weight set to this value (e.g. 0.05 = 5% of total cred).
+  // (Mostly included so we can have SourceCred receiving cred by default)
```

Comment:
This isn't precisely correct. a weight of 0.05 mints an additional 5% of base contributor cred, which means that it will be 0.05 / 1.05 of total cred, or about 4.8%